tests/unit/remoteworker/test_remote_worker.py

```diff
@@ -141,3 +143,69 @@ def test_return_url_per_environment(self):
         expected_url = "http://testserver:80{}".format(url)
         returned_url = return_url_per_environment(url)
         self.assertEqual(returned_url, expected_url)
+
+
+@mock.patch("scripts.workers.remote_submission_worker.get_challenge_by_queue_name")
+@mock.patch("scripts.workers.remote_submission_worker.create_dir_as_python_package")
+@mock.patch("scripts.workers.remote_submission_worker.extract_challenge_data")
+class LoadChallengeTestClass(BaseTestClass):
+    def setUp(self):
+        super(LoadChallengeTestClass, self).setUp()
+        self.user = {"username": 'someuser', "email": "user@test.com"}
+
+        self.challenge_host_team = {
+            "team_name": 'Test Challenge Host Team',
+            "created_by": self.user,
+        }
+
+        self.challenge = {
+            "id": self.challenge_pk,
+            "title": 'Test Challenge',
+            "description": 'Description for test challenge',
+            "terms_and_conditions": 'Terms and conditions for test challenge',
+            "submission_guidelines": 'Submission guidelines for test challenge',
+            "creator": self.challenge_host_team,
+            "published": True,
+            "approved_by_admin": True,
+            "enable_forum": True,
+            "anonymous_leaderboard": False,
+        }
+
+        self.challenge_phase = {
+            "id": self.challenge_phase_pk,
+            "name": 'Challenge Phase',
+            "description": 'Description for Challenge Phase',
+            "leaderboard_public": False,
+            "is_public": True,
+            "challenge": self.challenge
+        }
+
+        self.phases = [self.challenge_phase]
+
+        self.queue_name_patcher = mock.patch(
+            "scripts.workers.remote_submission_worker.QUEUE_NAME", "evalai_submission_queue"
+        )
+        self.queue_name_patcher.start()
+
+    def tearDown(self):
+        self.queue_name_patcher.stop()
+        # super(LoadChallengeTestClass, self).tearDown()  # FUTURE
```

Comment:
@nikochiko if we don't need this can we remove it?

Reply:
Sure! It's not required. I just added it so it isn't forgotten. Will update :+1: